### ISSUE TITLE
Add a `pixi project feature` to add, remove, list, and edit features.

### ISSUE THREAD

**Issue by zbowling** [prefix-dev/pixi#1999]
### Problem description

Using the CLI, it's possible to add, remove, list, and edit environments, but you can't do the same for features. 

You can only manage features by editing the manifest file directly, but you can associate an environment to a feature in with the pixi project environments add <env> --features <features> flag.

**Comment by zbowling** [comment]
It looks like `pixi task add --feature test test "pytest"` will implicitly create a feature if it doesn't exist which is interesting.

**Comment by tdejager** [comment]
So I think you would like an extra CLI under `pixi project`? :)

**Comment by zbowling** [comment]
Yes. Just rounding out some of the things you can use the CLI to edit in the toml. Nearly everything is accessible except “features”.

**Comment by tdejager** [comment]
Got it! We should actually also add the `pypi-options` there :)
